plugin.video.orftvthek/default.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/python
```

Comment:
i'd be nice if you could turn the entrypoint as a simple dummy file which imports the main addon file and simply runs it:

```
from resources.lib import myaddon
myaddon.run()
```

Reasoning is that this is the only file that is not compiled to bytecode by the python interpreter

Reply:
that totally makes sense. fixed in the new commit